conda.yaml

```diff
@@ -5,4 +5,4 @@ dependencies:
   - python=3.7.5
   - pip=20.1
   - pip:
-      - rpaframework==11.2.1
+     - rpaframework-11.3.0.245.6fa8dc6-py3-none-any.whl
```

Comment:
You can use now the latest [devpi](https://devpi.robocorp.cloud/ci/dev) build like so: `https://robocorp:freeasinbeer@devpi.robocorp.cloud/ci/dev/+f/dbb/d083ad337f24b/rpaframework-11.3.0-py3-none-any.whl#sha256=dbbd083ad337f24bcb3214e18136e9a71927d7b77527024bb975b8b8e49fc3ff`.

If the changes aren't merged in master yet, then of course, you can build and place inside the wheel like you've done already or reference it from a RAW wheel GitHub URL (requires pushing it first).